elasticsearch/k8_util.py

```diff
@@ -48,7 +48,9 @@ def create_cluster(cluster_config):
       --enable-autoupgrade \
       --enable-autorepair \
       --max-surge-upgrade 1 \
-      --max-unavailable-upgrade 0
+      --max-unavailable-upgrade 0 \
```

Comment:
Do you think there is any good reason to make the use of public or private-only IP addresses config-driven?
I can't think of a good reason to have public ip addresses, so I'm fine with this just being the standard deployment.

Please update the README to note that the cluster is deployed without public IP addresses.

Reply:
I think its also fine for this to just be part of the standard deployment. 

README updated.